wpilib/wpilib/accumulatorresult.py

```diff
@@ -0,0 +1,16 @@
+#----------------------------------------------------------------------------
+# Copyright (c) FIRST 2008-2017. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+#----------------------------------------------------------------------------
+
+from collections import namedtuple
+
+__all__ = ['AccumulatorResult']
+
+AccumulatorResult = namedtuple('AccumulatorResult', ['value', 'count'])
```

Comment:
Hm. Would it make sense to type hint this (using `typing.NamedTuple`)?

Reply:
Knock yourself out.